client/js/widgets/widget.js

```diff
@@ -1846,10 +1862,11 @@ export class Widget extends StateManaged {
       await this.checkParent();
 
       const lastHoverTarget = this.hoverTarget;
-      const myCenter = center(this.domElement);
+      let anchor = this.dropAnchor();
+      let dropPoint = anchor ? screenCoordFromLocalPoint(this.domElement, anchor) : center(this.domElement);
```

Comment:
@96LawDawg this is a reason you might not like using drop anchor for grid alignment - because it is also changing the anchor point which is used to determine which holder a widget drops into. It probably still makes sense that drop anchor sets the default grid alignment point, but maybe we should have a property which is specifically for grid alignment that doesn't also change the anchor point for targeting holders.

Reply:
I played around with that a little.  I don't think that is going to be a problem.  Few games require grid widgets to use holders.  That's one of the advantages of grids is that you can pre-determine where things are going to be.  Also, for the games that do have grids and holders, there is generally some wiggle room and I don't think a slight hoverTarget difference will matter.

But if you think this is a concern, maybe there is an alternative.  Would it work to skip that section of the code if there is a grid active on the widget?  Or maybe that defeats something else you are trying to do.  Just a thought.

Reply:
The alternative I was thinking of (for now) is just having a property that if specified would be used instead of dropAnchor for the snap anchor. We could always add this later.